tools/make-v8.sh

```diff
@@ -1,38 +1,47 @@
 #!/bin/bash
 
-
-git_origin=$(git config --get remote.origin.url | sed 's/.\+[\/:]\([^\/]\+\/[^\/]\+\)$/\1/')
-git_branch=$(git rev-parse --abbrev-ref HEAD)
-v8ver=${1:-v8} #default v8
-svn_prefix=https://github.com
-svn_path="$svn_prefix/$git_origin/branches/$git_branch/deps/$v8ver"
-#svn_path="$git_origin/branches/$git_branch/deps/$v8ver"
-gclient_string="solutions = [{'name': 'v8', 'url': '$svn_path', 'managed': False}]"
+# Get V8 branch from v8/include/v8-version.h
+MAJOR=$(grep V8_MAJOR_VERSION deps/v8/include/v8-version.h | cut -d ' ' -f 3)
+MINOR=$(grep V8_MINOR_VERSION deps/v8/include/v8-version.h | cut -d ' ' -f 3)
```

Comment:
which should be the case always, since this script is only run by make and the Makefile is in that directory.